pkg/resources/deployment.go

```diff
@@ -0,0 +1,74 @@
+package resources
+
+import (
+	appsv1beta1 "k8s.io/api/apps/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Deployment(namespace string) *appsv1beta1.Deployment {
+	return &appsv1beta1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "ark",
+			Namespace: namespace,
+		},
+		Spec: appsv1beta1.DeploymentSpec{
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"component": "ark",
+					},
+				},
+				Spec: corev1.PodSpec{
+					RestartPolicy:      corev1.RestartPolicyAlways,
+					ServiceAccountName: "ark",
+					Containers: []corev1.Container{
+						{
+							Name:            "ark",
+							Image:           "gcr.io/heptio-images/ark:latest",
+							ImagePullPolicy: corev1.PullAlways,
+							Command: []string{
+								"/ark",
+							},
+							Args: []string{
+								"server",
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "cloud-credentials",
+									MountPath: "/credentials",
+								},
+								{
+									Name:      "plugins",
+									MountPath: "/plugins",
+								},
+							},
+							Env: []corev1.EnvVar{
+								{
+									Name:  "GOOGLE_APPLICATION_CREDENTIALS",
```

Comment:
We could add the AWS env var here too - no harm in having both. Eventually we'll remove them and make the go code set them if needed.

Reply:
Ok, is that the only thing that is different between the GCP and AWS configurations?

Reply:
Yes